.github/workflows/windows-ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: windows-ci
+
+on:
+  [push, pull-request]
+
+jobs:
+  compile:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.5', '3.6', '3.7', '3.8']
```

Comment:
I don't think there is any need compile-check on all supported Python version, our use of the Python CAPI is very basic and I don't expect any issue. Although, if it is fast I don't see a reason why to drop it...